Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1875,6 +1861,9 @@ private Contract CreateContract(Symbol symbol, bool includeExpired, string excha
                                         _futuresExchanges[symbol.ID.Market] :
                                         symbol.ID.Market;
 
+                var symbolProperties = _symbolPropertiesDatabase.GetSymbolProperties(symbol.ID.Market, contract.Symbol, SecurityType.Future, Currencies.USD);
```

Comment:
`contract.Symbol` is the IB mapped ticker, might not match the SPDB? We do have access to the `Symbol` instance here, maybe we could use it?